config/application.php

```diff
@@ -63,7 +63,7 @@
  * Custom Settings
  */
 define('AUTOMATIC_UPDATER_DISABLED', true);
-define('DISABLE_WP_CRON', getenv('DISABLE_WP_CRON') ?: false);
+define('DISABLE_WP_CRON', getenv('DISABLE_WP_CRON') == "true");
```

Comment:
@dvorakluk sorry to nitpick, but can you change these to single quotes? We use them for consistency.